django/thunderstore/repository/models/comment.py

```diff
@@ -0,0 +1,63 @@
+from django.conf import settings
+from django.contrib.auth.models import User as UserType
+from django.core.exceptions import PermissionDenied
+from django.db import models
+from ulid2 import generate_ulid_as_uuid
+
+from thunderstore.community.models import PackageListing
+from thunderstore.core.mixins import TimestampMixin
+from thunderstore.core.utils import capture_exception
+from thunderstore.repository.models.thread import CommentsThreadMixin, Thread
+
+
+class Comment(CommentsThreadMixin, TimestampMixin, models.Model):
+    thread = models.ForeignKey(
+        Thread,
+        on_delete=models.CASCADE,
+        related_name="comments",
+    )
+    author = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.PROTECT,
+        related_name="comments",
+    )
+    content = models.TextField(
+        max_length=2048,
+    )
+    uuid = models.UUIDField(
+        default=generate_ulid_as_uuid,
+        editable=False,
+        unique=True,
```

Comment:
IIRC it's redundant to set `unique` if it's set as the primary key, but I don't mind it existing either way if you want to keep it:

Quoting [the docs](https://docs.djangoproject.com/en/dev/ref/models/fields/#primary-key):
> primary_key=True implies null=False and unique=True. Only one primary key is allowed on an object.